tutorials/workflow/csharp/combined-patterns/WorkflowApp/Program.cs

```diff
@@ -0,0 +1,90 @@
+using Dapr.Client;
+using Dapr.Workflow;
+using WorkflowApp;
+using WorkflowApp.Activities;
+
+var builder = WebApplication.CreateBuilder(args);
+builder.Services.AddSingleton<HttpClient>(DaprClient.CreateInvokeHttpClient(appId: "shipping"));
+builder.Services.AddSingleton<InventoryManagement>();
+builder.Services.AddDaprClient();
+builder.Services.AddDaprWorkflow(options =>
+{
+    options.RegisterWorkflow<OrderWorkflow>();
+    options.RegisterActivity<CheckInventory>();
+    options.RegisterActivity<CheckShippingDestination>();
+    options.RegisterActivity<UpdateInventory>();
+    options.RegisterActivity<ProcessPayment>();
+    options.RegisterActivity<ReimburseCustomer>();
+    options.RegisterActivity<RegisterShipment>();
+});
+var app = builder.Build();
+app.UseCloudEvents();
+
+app.MapPost("/start", async (
+    Order order,
+    InventoryManagement inventory,
+    DaprWorkflowClient workflowClient) =>
+{
+
+    // This is to ensure to have enough inventory for the order.
+    await inventory.CreateDefaultInventoryAsync();
+
+    var instanceId = await workflowClient.ScheduleNewWorkflowAsync(
+        name: nameof(OrderWorkflow),
+        instanceId: order.Id,
+        input: order);
+
+    return Results.Accepted(instanceId);
+});
+
+app.MapPost("/shipmentRegistered", async (
```

Comment:
Same here on the comments. Since this is a tutorial we want to air on the side of comments. Like 

// This endpoint subscribes to the pubsub broker that is published on by the ShippingApp. It automatically raises an external event to  register the shipment